users/locale/fi/LC_MESSAGES/django.po

```diff
@@ -7,8 +7,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-07-16 14:08+0000\n"
-"PO-Revision-Date: 2020-07-16 17:09+0300\n"
+"POT-Creation-Date: 2020-09-26 13:25+0300\n"
+"PO-Revision-Date: 2020-09-26 13:40+0300\n"
 "Last-Translator: Tatu Wikman <tatu.wikman@gmail.com>\n"
```

Comment:
Might want to change these when updating translations :)

Reply:
So... POedit software isn't perfect either it seems if it doens't touch those on save... meh.. :)